README.md

```diff
@@ -1,25 +1,85 @@
-# PHP Version Information Page (codename `servehappy`)
-
+# WordPress PHP education initiative (codename `servehappy`).
 [![Edit in GitHub](https://img.shields.io/badge/Edit_in_GitHub--green.svg?style=social)](https://github.com/WordPress/servehappy/edit/master/README.md)
 
-**Information page about PHP versions and updates, to be used through the WordPress.org project.**
+The servehappy project is an initiative that seeks to educate WordPress site owners about the value of having the latest version of PHP powering their WordPress site.  This supports the overall goal of increasing percentage of WordPress installs running modern PHP versions.
 
-> Note: `servehappy` is only a temporary codename for now.
+## Background
 
-This is the main repository for working on the `servehappy` page(s). It is used to collect the discussions in the issue tracker and work on the page content.
+One of the core tenets of WordPress has been that of being backward compatible and the support of PHP 5.2 is and has been part and parcel of that tenet.  Over the years however, there has been increasing tension within the WordPress developer community on this support for PHP5.2 between those who support keeping the minimum PHP version requirements the same for WordPress core, and those who want to increase the minimum PHP version requirements for WordPress core.  There have been numerous arguments made for either side and you can follow some of the discussion [here](https://core.trac.wordpress.org/ticket/33381).
```

Comment:
I think there should be a space between `PHP` and `5.2` in the second time in the paragraph.

Reply:
Done in latest commit